plugin/cypress.config.ts

```diff
@@ -14,13 +14,21 @@ export default defineConfig({
   requestTimeout: 15000,
   responseTimeout: 15000,
   fixturesFolder: 'cypress/fixtures',
+  chromeWebSecurity: false,
+  screenshotsFolder: "results/screenshots",
+  videosFolder: "results/videos",
+  // videoUploadOnPasses: false, // this is not supported in cypress 13+, TODO add this back to config
+
   env: {
+    USERNAME: "jenkins", // default value for jenkins
+    OC_IDP: "my_htpasswd_provider", // default value for jenkins, can vary based on cluster setup
     'cypress-react-selector': {
       root: '#root'
     },
     omitFiltered: true,
     filterSpecs: true
   },
+
   e2e: {
     baseUrl: 'http://localhost:3000',
```

Comment:
Openshift CRC local base url is `http://localhost:9000`

Reply:
fixed